cmd/skopeo/copy.go

```diff
@@ -260,6 +261,15 @@ func (opts *copyOptions) run(args []string, stdout io.Writer) (retErr error) {
 		}
 	}
 
+	if destRef.Transport().Name() != "dir" {
```

Comment:
`destRef.Transport().Name() != c/image/v5/directory.Transport.Name()` please, to make this dependency visible to IDEs.